{
    "type": "bundle",
    "id": "bundle--92951e43-c8cb-4f8c-92ba-fae94064be20",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--e465358d-c8df-4b41-a6c9-2a07a4f6767b",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-01-29T00:36:37.968687Z",
            "modified": "2025-01-29T00:36:37.968687Z",
            "name": "CVE-2025-24478",
            "description": "A denial-of-service vulnerability exists in the affected products. The vulnerability could allow a remote, non-privileged user to send malicious requests resulting in a major nonrecoverable fault causing a denial-of-service.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-24478"
                }
            ]
        }
    ]
}